pipelines/rbsa/tasks/zipcode_correlation.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import pandas as pd
+import numpy as np
+from generics import task as t
+from generics.file_type_enum import SupportedFileReadType
+
+
+logger = logging.getLogger('LCTK_APPLICATION_LOGGER')
+
+
+class ZipcodeCorrelation(t.Task):
+    def __init__(self, name):
+        super().__init__(self)
+        self.name = name
+
+        self.input_artifact_full_zipcodes = 'full_zipcodes.csv'
+        self.input_artifact_projection_locations = 'PROJECTION_LOCATIONS.json'
+
+        # these will be used to generate list of input files
+        self.pre_data_files = [ 
+            { 'name': self.input_artifact_full_zipcodes, 'read_type': SupportedFileReadType.DATA },
+            { 'name': self.input_artifact_projection_locations, 'read_type': SupportedFileReadType.CONFIG }
+        ]
+
+        self.task_function = self._task
+        self.output_artifact_correlation_matrix = 'correlation_matrix.csv'
+        
+        self.data_map = None
+        self.df = None
+
+    def _save_data(self):
+        self.save_data(self.df)
```

Comment:
looks like this function isn't being used or called from anywhere - I think we can just remove it

Reply:
fixed